internal/hud/server/server.go

```diff
@@ -239,6 +240,10 @@ func (s *HeadsUpServer) HandleSetTiltfileArgs(w http.ResponseWriter, req *http.R
 	}
 }
 
+// Responds with:
+// * 200/empty body on success
+// * 200/error message in body on well-formed, unservicable requests (e.g. resource is disabled or doesn't exist)
```

Comment:
seems like we should do 404 in this case? that's probably what k8s would do (e.g., trying to exec on a pod that didn't exist). (404s can still have error messages)